### ISSUE TITLE
No Issue just thanks

### ISSUE THREAD

**Issue by SuloxyA500** [hoglet67/RGBtoHDMI#182]
If this post does not belong here, please delete it.

I just want to thank you for your work.
I installed it on the pi zero and connected it to my A500.
It works like a charme, right out of the box, no problems at all, even on my 4k display!

Best Regards
Suloxy

**Comment by abrugsch** [comment]
Same here. Much thanks for this great project!

**Comment by bluearcus** [comment]
Absolutely... this has transformed the output options for several of my machines already, and more still to come!

**Comment by bkodenkt** [comment]
Absolute game changer for connecting my Amigas to current TVs and displays. Thank for this!

**Comment by jdryyz** [comment]
Agreed! Fantastic project.

**Comment by jdryyz** [comment]
I have to add that I also discovered that my A2000 RGB2HDMI system was two versions behind. Now that I have updated to the latest, the image quality is even better! It solved something I was going to ask about and then added features I was interested in as well. So awesome!
